mock/mock.py

```diff
@@ -2059,8 +2059,16 @@ def _format_call_signature(name, args, kwargs):
     message = '%s(%%s)' % name
     formatted_args = ''
     args_string = ', '.join([repr(arg) for arg in args])
+
+    if six.PY2 and isinstance(key, unicode):
```

Comment:
I guess we can move this check into the `encode_item` function in order to fix the UnboundLocalError error (see Travis CI)